runtime/pkg/rilltime/rilltime.go

```diff
@@ -96,75 +106,94 @@ var (
 		timeutil.TimeGrainHour:    timeutil.TimeGrainMinute,
 		timeutil.TimeGrainDay:     timeutil.TimeGrainHour,
 		timeutil.TimeGrainWeek:    timeutil.TimeGrainDay,
-		timeutil.TimeGrainMonth:   timeutil.TimeGrainWeek,
+		timeutil.TimeGrainMonth:   timeutil.TimeGrainDay,
```

Comment:
unintentional?

Reply:
Same here, we dont implicitly use week. `lowerOrderMap` is used to return the default grain when the duration is exactly 1 month. EG: `-2M to -1M` should default to day grain.